coral/config.json

```diff
@@ -34,9 +34,13 @@
         "product/priv-app/qcrilmsgtunnel/qcrilmsgtunnel.apk",
         "product/framework/QtiTelephonyServicelibrary.jar",
         "product/framework/com.qualcomm.qti.uceservice-V2.1-java.jar",
-        "product/framework/qcrilhook.jar"
+        "product/framework/qcrilhook.jar",
+        "product/framework/qti-telephony-hidl-wrapper.jar",
+        "product/framework/qti-telephony-utils.jar"
```

Comment:
Turns out we have a caf repo for other caf devices (like jasmine_sprout), which includes these.

See https://gitlab.com/calyxos/vendor_codeaurora_telephony